### DISCUSSION TITLE
Restrict Admin access

### DISCUSSION THREAD

**Discussion by ASLLR** [YOURLS/YOURLS#3637] [Q&A] [answered]
Hello everyone, 

i would like to know a thing about the use of yourls. 

I would like to install the yourls service in my company and open it from the internet. However, I would like users to need to log in to use the URL shortener service. But I don't want all users who log in to have access to the administration interface.
In YOURLS do we have the possibility of having a mandatory connection and splitting access rights to the administration panel?

Thank in advance,

**Comment by dgw** [comment, marked as answer]
Possibly something like https://github.com/joshp23/YOURLS-AuthMgrPlus will do what you want. I'm not 100% sure I understand the exact use case requested, but I'm positive that YOURLS by itself, with no plugins, can't do it.

**Reply by ASLLR** [reply]
Hi @dgw ! Thank your for your answer, i think this can be a solution ! I try this thank you :)
